static/css/main.css

```diff
@@ -44,6 +44,11 @@ ul {
   width: 40px;
 }
 
+.twitter > iframe {
+  text-align: center;
+  width: 100%;
```

Comment:
Try `100% !important` 

It's being overridden by the inline styling :/
if you check the devtool (inspect element)